app/index.js

```diff
@@ -482,7 +482,7 @@ app.on('ready', () => {
         title: 'Save password?',
         message: message,
         buttons: ['Yes', 'No'],
-        defaultId: 0,
+        defaultId: 1,
```

Comment:
If someone has your computer for 1 minute, they can install a rootkit on it. They can steal any cookies from sites where you are logged in. They can also open Chrome and get your Chrome passwords from the Chrome built-in password manager.

Removing default focus from a button when the user has chosen "Save passwords" in preferences is just inconvenient. However, I think the issue you are addressing (user accidentally saving a password) will be fixed when the password prompt UI becomes a notification bar instead of a dialog anyway.